src/user-management/interfaces/user-management-factor.interface.ts

```diff
@@ -0,0 +1,21 @@
+import { Totp, TotpResponse } from '../../mfa/interfaces/totp.interface';
+
+export interface UserManagementFactor {
```

Comment:
now that these are in `user-management`, what do you think about dropping the `UserManagement` prefix from the names?

Reply:
yeah that seems reasonable

Reply:
done